src/main/java/com/faforever/client/mod/ModManagerController.java

```diff
@@ -86,6 +97,23 @@ public void initialize() {
     loadActivatedMods();
 
     modListView.scrollTo(modListView.getSelectionModel().getSelectedItem());
+
+    modSearchTextField.textProperty().addListener(new ChangeListener<String>() {
+      @Override
+      public void changed(ObservableValue<? extends String> observable, String oldValue, String newValue) {
+        if (newValue.isEmpty()) {
+          modVersionFilteredList.setPredicate(viewToggleGroup.getSelectedToggle() == uiModsButton ? UI_FILTER : SIM_FILTER);
+        } else {
+          modVersionFilteredList.setPredicate(getCombinedFilter(newValue));
+        }
+      }
+    });
+    EventHandler keyEventHandler = new SearchTextFieldKeyEventHandler(modListView,modVersionFilteredList);
+    modSearchTextField.setOnKeyPressed(keyEventHandler);
```

Comment:
Probably don't want this bound because of the multi select model

Reply:
removed